async_upnp_client/client.py

```diff
@@ -421,6 +479,64 @@ async def async_call_action(
         result = await action.async_call(**kwargs)
         return result
 
+    def add_subscriber(self, subscriber: EventSubscriber) -> None:
```

Comment:
I am wondering whether the subscription handling should be in `client.py` or in `server.py`.